types/array.go

```diff
@@ -519,13 +519,13 @@ func (a *BoolArray) scanBytes(src []byte) error {
 	} else {
 		b := make(BoolArray, len(elems))
 		for i, v := range elems {
-			if len(v) != 1 {
+			if len(v) < 1 {
```

Comment:
These fixes seem generic, should they make their way into `master` earlier than the v3 code?

Reply:
Yep, will do.